server/src/main/java/org/apache/cassandra/sidecar/config/KeyStoreConfiguration.java

```diff
@@ -55,7 +55,7 @@ public interface KeyStoreConfiguration
      */
     default boolean reloadStore()
     {
-        return checkInterval().compareTo(SecondBoundConfiguration.ZERO) > 0;
+        return checkInterval() != SecondBoundConfiguration.ZERO;
```

Comment:
maybe compare by value, and not reference?
```suggestion
        return !checkInterval().equals(SecondBoundConfiguration.ZERO);
```